internal/pkg/cli/deploy/svc.go

```diff
@@ -1171,17 +1171,17 @@ func (d *workerSvcDeployer) stackConfiguration(in *StackRuntimeConfiguration) (*
 	for _, topic := range topics {
 		topicARNs = append(topicARNs, topic.ARN())
 	}
-	subs := d.wsMft.Subscriptions()
+	subs := d.wsMft.Subscribe.Subscriptions()
```

Comment:
nit: reading `d.wsMft.Subscribe.Subscriptions()` is odd (`.Subscribe` should already be the subscriptions), can we move it back to be at the manifest level?